README_EN.md

```diff
@@ -0,0 +1,233 @@
+
+Rexxar iOS supports iOS 7.0 and above systems.
+
+
+## Rexxar
+
+About Rexxar's integral introduction, please check this article: [豆瓣的混合开发框架 -- Rexxar](http://lincode.github.io/Rexxar-OpenSource). Sorry about these is only Chinese version at present.
+
+Rexxar includes threes libraries:
+
+- Rexxar Web ：[https://github.com/douban/rexxar-web](https://github.com/douban/rexxar-web)。
+- Rexxar Android：[https://github.com/douban/rexxar-android](https://github.com/douban/rexxar-android)。
+- Rexxar iOS：[https://github.com/douban/rexxar-ios](https://github.com/douban/rexxar-ios)。
+
+
+## Installation
+
+### Install Cocoapods
+
+[CocoaPods](http://cocoapods.org) is a dependency manager for Objective-C and Swift. You can install it with the following command:
+
+```bash
+$ gem install cocoapods
+```
+
+### Podfile
+
+```ruby
+target 'TargetName' do
+  pod 'Rexxar', :git => 'https://github.com/douban/rexxar-ios.git', :commit => '0.2.1'
+end
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+
+## Usage
+
+You can take a look at the example in the RexxarDemo folder.
+
+We use github raw file to serve the routes map api for this demo. You definitely need implement a real server in your real product serving the routes map api and html, css, javascript resource files. 
+
+You can use any server framework. Rexxar hasn't any requirement about the server framework. Rexxar offers the configuration interface of the routes map api address in `RXRConfig`. You can find the way to configure the address in next section.
+
+### Configure routes map api address
+
+```Swift
+  RXRConfig.setRoutesMapURL(NSURL(string:"https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/routes.json)!)
+```
+
+Rexxar use url to identify the page in mobile application. With a valid url, we can create a RXRViewController. There is the map from url to html resources in the routes map api file. In the Demo, you can see a routes map api file like this:
+
+```json
+{
+    "items": [{
+        "remote_file": "https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/rexxar/demo-252452ae58.html",
+        "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+        "uri": "douban://douban.com/rexxar_demo[/]?.*"
+    }],
+    "partial_items": [{
+        "remote_file": "https://raw.githubusercontent.com/douban/rexxar-web/master/example/dist/rexxar/demo-252452ae58.html",
+        "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+        "uri": "douban://partial.douban.com/rexxar_demo/_.*"
+    }],
+    "deploy_time": "Sun, 09 Oct 2016 07:43:47 GMT",
+}
+```
+
+### Configure the resource path
+
+```Swift
+  RXRConfig.setRoutesResourcePath("rexxar")
+```
+
+Using Rexxar to develop mobile application, we usually put a copy of resource files in the application's release package. This copy of resource files in app's package will reduce user's download of resource files and optimise the user's experience of application.
```

Comment:
We usually ship a copy of HTML resource files with the application's release package, in order to boost the initial load of the hybrid page. You can set the local path to the folder that contains the resource files with this API. Please be reminded that the folder type in Xcode must be set as `folder reference`, with a blue folder icon; instead of the `group` type with a yellow icon.